ci/unreal-engine.version

```diff
@@ -1 +1,2 @@
 HEAD 4.22-SpatialOSUnrealGDK
+HEAD 4.23-SpatialOSUnrealGDK-merge
```

Comment:
I think remove this for now - just CI against 4.22-SpatialOSUnrealGDK, and we will do the CI against two versions in the 4.23 branch